common/wireaddr.h

```diff
@@ -26,14 +26,15 @@ enum wire_addr_type {
 	ADDR_TYPE_PADDING = 0,
 	ADDR_TYPE_IPV4 = 1,
 	ADDR_TYPE_IPV6 = 2,
+	ADDR_TYPE_LOCAL_SOCKET = 5
```

Comment:
I am hesitant to put this in `struct wireaddr` as this is also used in BOLT standard-compliant `node_announcement` that get broadcast as node gossip.

At minimum the numbering should not be directly after the BOLT-defined ones.

It might be better to bring up this possible use case in lightning-dev mailing list.

Alternatively we could skip this address type when sending gossip announcements.

Reply:
Agreed, it makes no sense at all to broadcast these. It hadn't occurred to me. I will look into alternatives.

Reply:
I don't think there's much value in adding local sockets to the BOLTs but I settled on a MIME type (`application/lightning`) and an application id (`LIGHTNING`) for NFC payments. I'll bring this up on the ML when I get the time since I'm not sure how to proceed on standardising these.

Reply:
If we never broadcast these there is no need to bring it up on the mailinglist, but, I am uncertain if we should be using `struct wireaddr` then, or some other mechanism.  Intent, as I understand it,  is that `struct wireaddr`, is for standard broadcastable messages.  @rustyrussell @cdecker opinions?